docs/writing-docs/skillmaps.md

```diff
@@ -0,0 +1,222 @@
+
+![Full skillmap graphic](/static/skillmaps/skillmap.jpg)
+
+### Path properties
+
+The path properties are in the second-level (`##`) headings using the paths's name, or its _path-id_.
+
+* **path-id**: the identification name of the path
+
+```
+## space
+```
+
+After the heading is the path property list.
+
+* **name**: path display name
+* **description**: the path description
+* **completionUrl**: the URL path to a completion document (i.e. the "Certificate of Completion")
+
+Here's an example of a property list set for a path:
+
+```
+## space
+
+* name: Design a Space Explorer
+* description: Let's explore the depths of space! We'll design a vessel for space travel, add some enemies, and populate the universe with planets.
+* completionUrl: /static/skillmap/certificates/design-a-space-explorer.pdf
+```
+
+### ~ hint
+
+#### Selecting a path
+
+When selecting a path in a skillmap, the focus goes to the activity node. That node's properties are displayed along with the path description.
+
+### ~
+
+The selected path shows in the skillmap panel along with the property info from the activity node in focus.
+
+![Individual skillmap path](/static/skillmaps/skillmap-path.jpg)
+
+### Activities (nodes)
+
+An activity node sets a current activity and connects it to the next one. Activities are created as [tutorials](/writing-docs/tutorials) and linked together in the node properties.
+
+The activity node properties are under a third-level (`###`) heading which has the nodes's name, or its _node-id_. The activity nodes are grouped sequentially under their associated path section.
+
+* **node-id**: the identification name of the node
+
+```
+### space-activity3
+```
+
+The node properties are specified under the `node-id` heading.
+
+* **name**: the display name of the activity node
+* **description**: description of the activity
+* **type**: the type of node, `tutorial` or `certificate`. Currently only the `tutorial` type is used
+* **tags**: a tag list to categorize the activity, like: `easy, enemies`
+* **next**: the `node-id` of the next activity
+* **url**: the URL for the tutorial document
+* **imageUrl**: a url for an image to display in the panel: jpg, png, or gif
+* **allowcodecarryover**: this is `true` by default, you can turn it off by setting it to `false`
+* **kind**: `layout` or `completion` (reserved, don't use)
+
+The node with it's properties will look like this:
+
+```
+### space-activity3
+
+* name: Enemies
+* description: Watch out for danger! Add enemies and lives to your game.
+* type: tutorial
+* tags: easy, enemies, kinds
+* next: space-activity4
+
+* url: /skillmap/space/activity3
+* imageUrl: /static/skillmap/space/enemies.gif
+```
+
+The node `name`, `description`, and `tags` are shown in the skillmap panel.
+
+![skillmap node](/static/skillmaps/skillmap-node.jpg)
+
+### Rewards
+
+The reward node is set automatically at end of the path if the path's `completionUrl` property is set.
```

Comment:
we didn't end up doing the automatic inference thing, so content creators have to specify a node with the `completion` kind to render a trophy/reward node